cypress/support/firestore_map_bounds.js

```diff
@@ -9,7 +9,8 @@ export {assertFirestoreMapBounds, expectLatLngBoundsWithin};
  *     expectedLatLngBounds
  */
 function assertFirestoreMapBounds(expectedLatLngBounds) {
-  cy.wrap(readDisasterDocument()).then((doc) => {
+  // Make sure we don't call readDisasterDocument until Cypress is ready.
+  cy.wrap(null).then(readDisasterDocument).then((doc) => {
```

Comment:
cypress was executing the code inside cy.wrap before cypress was ready?

Reply:
Yeah, remember that Cypress has no superpowers. The Javascript interpreter still executes the code, and Cypress just makes sure that its internal commands don't execute until certain conditions are met. Thus, cy.wrap(blah()) will execute blah() and pass the result to cy.wrap. Cypress can then make sure a later cy.get() doesn't do anything until the blah result completes (because cy.get() itself is just putting a command on the Cypress queue to execute later), but it can't control blah() executing.